src/cmd/root.go

```diff
@@ -16,8 +16,26 @@
 
 package cmd
 
+/*
+#cgo LDFLAGS: -l subid
+#include <shadow/subid.h>
+#include <stdlib.h>
+#include <stdio.h>
+const char *Prog = "storage";
+FILE *shadow_logfd = NULL;
+struct subid_range get_range(struct subid_range *ranges, int i)
+{
+	shadow_logfd = stderr;
+	return ranges[i];
+}
+#if !defined(SUBID_ABI_MAJOR) || (SUBID_ABI_MAJOR < 4)
```

Comment:
Nitpick: how important is to support the older `libsubid`?

I am asking because were testing on RHEL 8 and it has shadow-utils-4.6.  I see that even Debian 9 (stretch) seems to have [shadow-utils-4.4](https://packages.debian.org/source/stretch/shadow).  Isn't that new enough?

I know that [github.com/containers/storage/pkg/idtools](https://github.com/containers/storage/blob/main/pkg/idtools/idtools_supported.go) has this, but maybe it's not needed anymore?

Reply:
I only have it here because I copied it from idtools. Like you said, shadow-utils 4 has been around for a long time now.  I'm OK with removing it.

Reply:
Git forensics says that the ABI was [changed](https://github.com/shadow-maint/shadow/commit/32f641b207f6ddff1460a22fdc654def4a9c473d) in release 4.10.  That means that the older `libsuid` ABI is still relevant.